tests/builders/test_builders.py

```diff
@@ -138,3 +138,12 @@ def test_file_count():
         configuration=config,
     )
     cmd.build()
+    
+def test_regex_check():
+    """This test buildspec using status check with  'status_regex'"""
+    cmd = BuildTest(
+        buildspecs=[os.path.join(here, "status_regex.yml")],
+        buildtest_system=system,
+        configuration=config,
+    )
+    cmd.build()
```

Comment:
this looks good. Take a look at https://app.codecov.io/gh/buildtesters/buildtest/pull/1497/blob/buildtest/buildsystem/checks.py 

In particular the missing coverage for the else statement. Can you please try to add coverage for this. You will want to create a custom buildspec and just simply run it or you can update the main `status_regex.yml`
![Screen Shot 2023-06-23 at 9 28 08 AM](https://github.com/buildtesters/buildtest/assets/12942230/45c6897a-a6e0-4300-a658-c20b91a27916)